src/components/views/rooms/MessageComposerButtons.tsx

```diff
@@ -84,7 +85,7 @@ const MessageComposerButtons: React.FC<IProps> = (props: IProps) => {
         moreButtons = [
             showStickersButton(props),
             voiceRecordingButton(props, narrow),
-            pollButton(room, props.relation),
+            pollButton(room, props.relation, props.showPollsButton),
```

Comment:
feels more legible to me for the filtering out to happen here or in `pollButton()`, rather than in the `PollButton` component